mne/io/brainvision/brainvision.py

```diff
@@ -64,12 +69,14 @@ class RawBrainVision(_BaseRaw):
     @verbose
     def __init__(self, vhdr_fname, montage=None,
                  eog=('HEOGL', 'HEOGR', 'VEOGb'), misc=(),
-                 scale=1., preload=False, response_trig_shift=0, verbose=None):
+                 scale=1., preload=False, response_trig_shift=0,
```

Comment:
I agree. didn't really like this argument implementation. with `brainvision`, the stimulus values and response values are represented up to a byte (1-256). we could have by default that the response trigger are also += 256. I think that this is triggering scheme is `brainvision` specific. I think this would clean up the arguments and maintain across-device consistency.